consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeCoordinator.java

```diff
@@ -439,7 +443,7 @@ public ForkchoiceResult updateForkChoice(
 
     if (newHead.getNumber() < blockchain.getChainHeadBlockNumber()
         && isDescendantOf(newHead, blockchain.getChainHeadHeader())) {
-      LOG.info("Ignoring update to old head");
+      debugLambda(LOG, "Ignoring update to old head {}", newHead::toLogString);
```

Comment:
"Ignoring head update {}", newHead

?

Reply:
we can be specific here, since we know that FcU is trying to set the head to an old block in the same chain of the current head and so we ignore it

Reply:
I was just wondering if it could be confusing that it says "old head" but what is inserted is the newHead

Reply:
what about renaming `newHead` to something like `tentativeNewHead` or `possibleNewHead` to state that it is not yet confirmed?